README.md

```diff
@@ -69,6 +69,7 @@ To install a package into the virtual environment:
 uv pip install flask                # Install Flask.
 uv pip install -r requirements.txt  # Install from a requirements.txt file.
 uv pip install -e .                 # Install the current project in editable mode.
+uv pip install package@file://.     # Install the current project from disk
```

Comment:
I think `file://` shouldn't be needed, should we omit it?

Reply:
oh i hadn't tried that yet , it works !